src/style.css

```diff
@@ -0,0 +1,146 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  margin: 0;
+  width: 198px;
+  height: 406px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.stars {
+  padding-left: 16px;
+  padding-top: 16px;
+  display: flex;
+}
+
+.stars__reviews {
+  margin: 0;
+  padding-right: 16px;
+  margin-left: 17px;
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+  flex: none;
+  width: 53px;
+  height: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.stars-star {
+  background: url(images/star.svg) center no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars-star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--4 .stars-star:nth-child(-n + 4) {
```

Comment:
it should work not only with 4

please, proofread your code with checklist and task requirements
<img width="736" alt="image" src="https://user-images.githubusercontent.com/102411130/234196841-8ce11c24-0565-4eef-9dae-be97fe09cf7f.png">